data_reports/128_WorldwideSCND.Rmd

```diff
@@ -1,17 +1,12 @@
 # Worldwide soil carbon and nitrogen data
 
-The `Worldwide soil carbon and nitrogen data` data set in ISCN3 contains XX layer-level rows and YY profile-level information rows after cleaning for ISCN3.5.
-Rows were/weren't removed from ISCN3 because of reasons.
-Columns were/weren't removed from ISCN3 because of reasons.
-Values in columns were reassigned to XX because of reasons.
-
 The `Worldwide soil carbon and nitrogen data` data set in ISCN3 contains `r nrow(dataset_layer)` layer-level information and `r nrow(dataset_profile)` profile-level information after cleaning for ISCN3.5.
```

Comment:
These `nrow` references won't compile in the final document because you haven't created the `dataset_layer` or the `dataset_profile` yet. While I like your reaction to reach for a bit of code instead of writing out the row numbers here, in this case you do need to have numerical values entered here.

Reply:
should I just remove the nrow and say ... "contains 'dataset_layer' ... and 'dataset_profile'"?

Reply:
Pull request is ready.